docs/checks/internal/checks.yaml

```diff
@@ -326,8 +326,7 @@ checks:
 
       The check works by looking at the authors of recent commits
       and checking the `Company` field on the GitHub user profile. A contributor
-      must have at least 5 commits in the last 30 commits. The check succeeds if
-      all contributors span at least 2 different organizations (companies).
+      must have at least 5 commits in the last 30 commits.
```

Comment:
maybe say `a user is considered a contributor if they have at least 5 commits in the last 30 commits.`

Reply:
note: the check currently gives the highest score if there are at least 3 contributors in the last 30 commits. Any number below that is score propportionally.

Reply:
I'll just fix the text for now. #1024 can be handled separately.